test/quantization/eager/test_quantize_eager_qat.py

```diff
@@ -340,6 +343,37 @@ def checkQuantized(model):
                 model = quantize_qat(model, test_only_train_fn, [self.img_data_2d_train])
                 checkQuantized(model)
 
+    @skipIfNoXNNPACK
+    def test_conv_linear_symm(self):
```

Comment:
do we need to add a test for numerics as well?

Reply:
Added! Thanks!